packages/frontend/src/components/sign/SignTransferRetry.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import ArrowUpImage from '../../images/icon-arrow-up-green.svg';
+import RetryImage from '../../images/icon-retry-tx.svg';
+import FormButton from '../common/FormButton';
+
+const CustomContainer = styled.div`
```

Comment:
@marcinbodnar can we please use the `/common/Container.css` as base wrapper for the page container like we do here: https://github.com/near/near-wallet/blob/a1fc1fe4bc362a23c74f8e52bcd36852791d099d/packages/frontend/src/components/login/v2/ConfirmLogin.js#L38

Reply:
You can also use it if you're creating a new styled component wrapper, like here: https://github.com/near/near-wallet/blob/a1fc1fe4bc362a23c74f8e52bcd36852791d099d/packages/frontend/src/components/send/SendContainerV2.js#L27

Reply:
Done.